interface/main/finder/patient_select.php

```diff
@@ -192,8 +192,9 @@ function submitList(offset) {
     }
 
     $sql = "SELECT $given FROM patient_data " .
-    "WHERE $where ORDER BY $orderby LIMIT $fstart, $sqllimit";
-    $rez = sqlStatement($sql, $sqlBindArray);
+    "WHERE $where ORDER BY $orderby LIMIT ? , $sqllimit";
```

Comment:
Sorry I didn't catch this before. You can't use sql binding on limits; this code will not work. But we have a function, just for this situation :)
https://github.com/openemr/openemr/blob/master/library/formdata.inc.php#L49
This function escape_limit() is available in every script and you can use it on both $fstart and $sqllimit above:
``````
"WHERE $where ORDER BY $orderby LIMIT " . escape_limit($fstart) . ", " . escape_limit($sqllimit);

Reply:
Done.